### PR TITLE
[DJFS-111] Develop a customer details list view page

### PR DESCRIPTION
**Pull request by Cynthia-Berry**
## Task: `[DJFS-111] Develop a customer details list view page`
- `Created database, tables and inserted records to the table using PHP. I also saved the inserted records in a cookie and displayed same records on the browser, these were done using PHP`

### CODE REVIEW

File: submissions/cynthiaNwakaeme/php-database/connection.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+     //connecting to mysql database
+
+     try {
+        $pdo = new PDO('mysql: host=localhost', 'root', 'mysql');
```
Comment (adaugochi) [inline submissions/cynthiaNwakaeme/php-database/connection.php:6]: How are you passing the database name here? I can only see hostname, username and password

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/cynthiaNwakaeme/php-database/connection.php (hunk 1 above)
Comment (Cynthia-Berry) [reply submissions/cynthiaNwakaeme/php-database/connection.php:6]: I used the syntax "Use db_name"(SQL SYNTAX) in the file where I am creating the database and table

File: submissions/cynthiaNwakaeme/php-database/cookies.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?php
+require_once('connection.php');
+
+echo "<h2> List of Customers at Taries Empire </h2>";
+
+try{
+    /* ACCESSING E-COMMERCE DATABASE AND QUERYING THE DATABASE*/
+    $use_db = "USE taries_empire";
```
Comment (Cynthia-Berry) [inline submissions/cynthiaNwakaeme/php-database/cookies.php:8]: I passed the database name Here.

Comment (adaugochi) [reply submissions/cynthiaNwakaeme/php-database/cookies.php:8]: Cool

**Review by adaugochi — APPROVED** [review]

**Pull request merged**